core-bundle/tests/Controller/ContentElement/ElementGroupTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Tests\Controller\ContentElement;
+
+use Contao\ContentModel;
+use Contao\CoreBundle\Controller\ContentElement\AccordionController;
+use Contao\CoreBundle\Fragment\Reference\ContentElementReference;
+use Symfony\Component\HttpFoundation\Request;
+
+class ElementGroupTest extends ContentElementTestCase
+{
+    public function testOutputsAccordion(): void
+    {
+        $model = $this->mockClassWithProperties(ContentModel::class, [
+            'id' => 1,
+            'type' => 'text',
+            'sectionHeadline' => 'Section',
+            'text' => '<p>Text.</p>',
+        ]);
+
+        $request = new Request();
+        $request->attributes->set('nestedFragments', [new ContentElementReference($model, 'main', [], true)]);
+
+        $response = $this->renderWithModelData(
+            new AccordionController($this->getDefaultFramework()),
+            [
+                'type' => 'element_group',
+            ],
+            null,
+            false,
+            $responseContextData,
+            null,
+            $request,
+        );
+
+        $expectedOutput = <<<'HTML'
+            <div class="content-element_group">
```

Comment:
I’d prefer not using underscores here.
```suggestion
            <div class="content-element-group">
```

Reply:
To accomplish this, we would have to adjust the `_base.html.twig` template, right?

https://github.com/contao/contao/blob/6983f1aed38d43d1c42b9e4c19bab800b27b1b90/core-bundle/contao/templates/twig/content_element/_base.html.twig#L12

Reply:
Or would you prefer to adjust the `AbstractContentElementController`?

```php
$classes[] = $this->generateCssClassFromType();
```

Reply:
Changed in 31853a796239d5347e24e0dc1e1d7cce4e991de8.

Reply:
Changed again in af908c3b36ff7cd32f94e91021c2283a7a1c6c88.